docs/guide/dev/howtos/basic_configuration/README.md

```diff
@@ -12,48 +12,48 @@ For licensing, see LICENSE.md.
 
 
-## How Do I Find the CKEditor Configuration Settings to Change?
+## How Do I Find the CKEditor 4 Configuration Settings to Change?
 
 A full listing of configuration settings that you can change in order to customize the editor to your needs can be found in the {@linkapi CKEDITOR.config CKEditor JavaScript API}. Use the methods described in the  {@link guide/dev/configuration/README Setting CKEditor Configuration} article from the Developer's Guide.
 
-Please note that not all configuration setting are available for each CKEditor build. Some of them are provided by plugins that may be missing in your editor installation. You can check which plugins are included in the official presets in this [comparison table](https://ckeditor.com/cke4/presets).
+Please note that not all configuration setting are available for each CKEditor 4 build. Some of them are provided by plugins that may be missing in your editor installation. You can check which plugins are included in the official presets in this [comparison table](https://ckeditor.com/cke4/presets).
 
 
-## How Do I Remove Unneeded CKEditor Functionality?
+## How Do I Remove Unneeded CKEditor 4 Functionality?
 
-CKEditor is a truly flexible tool with a modular structure — most editor functionality is based on plugins. Some core plugins are necessary for the editor to work or depend on one another, however, there are lots of optional plugins that you can skip when you do not need the functionality that they provide.
+CKEditor 4 is a truly flexible tool with a modular structure — most editor functionality is based on plugins. Some core plugins are necessary for the editor to work or depend on one another, however, there are lots of optional plugins that you can skip when you do not need the functionality that they provide.
 
 Remember that one of the most important {@link guide/dev/best_practices/README CKEditor best practices} is to customize your editor build with online builder and remove unneeded functionality before it makes it to your package. It is a bad practice to download the Full package and then remove plugins or buttons in your configuration. You will only be loading unnecessary stuff without any good reason. Check out the [4 Common CKEditor Installation Mistakes And How To Avoid Them](https://ckeditor.com/blog/4-Common-CKEditor-Installation-Mistakes-And-How-To-Avoid-Them) for a more thorough explanation of this practice.
 
-If you, however, want to disable some functionality that comes from a CKEditor plugin and is still in your build, you can use the {@linkapi CKEDITOR.config#removePlugins } setting to prevent the plugin from loading.
+If you, however, want to disable some functionality that comes from a CKEditor 4 plugin and is still in your build, you can use the {@linkapi CKEDITOR.config#removePlugins } setting to prevent the plugin from loading.
```

Comment:
```suggestion
If you, however, want to disable some functionality that comes from a CKEditor 4 plugin and is still in your build, you can use the {@linkapi CKEDITOR.config#removePlugins} setting to prevent the plugin from loading.
```